code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -227,9 +245,6 @@
 	I.req_gun_tags = list(GUN_PROJECTILE)
 	I.gun_loc_tag = GUN_MECHANISM
 
-/obj/item/gun_upgrade/underbarrel
```

Comment:
Why that was placed in between of `/muzzle` upgrades? This seems to be the right place.